master/static/srv/get_active_allocations_by_job.sql

```diff
@@ -0,0 +1,9 @@
+SELECT tasks.task_id AS TaskID,
+  tasks.job_id AS JobID,
+  SUM(case when allocations.state = 'RUNNING' then 1 else 0 end) AS NumRunning,
+  COUNT(allocations) AS NumStarting
+FROM tasks
+JOIN allocations ON allocations.task_id = tasks.task_id
+WHERE tasks.job_id in (SELECT unnest(string_to_array($1, ','))) -- Experiment match
+  AND allocations.state IN ('PULLING', 'RUNNING', 'STARTING')
+GROUP BY tasks.task_id, tasks.job_id;
```

Comment:
maybe something more like (table aliases, bool_or since we dont really need counts or sums, no group by task since this is for jobs, then the `enrichExperimentState` func can be simplified, too). also we have to join through jobs to get the index.
```
SELECT
  j.job_id,
  BOOL_OR(CASE WHEN a.state IN ('PULLING', 'STARTING') THEN true ELSE false END) AS is_starting,
  BOOL_OR(CASE WHEN a.state = 'RUNNING' THEN true ELSE false END) AS is_running
FROM jobs j
JOIN tasks t ON t.job_id = j.job_id
JOIN allocations a ON a.task_id = t.task_id
WHERE j.job_id in (SELECT unnest(string_to_array($1, ',')))
GROUP BY j.job_id;
```

Reply:
applied the SQL changes :+1: